docs/source/requirements.txt

```diff
@@ -1,2 +1,3 @@
+m2r2==0.2.7
```

Comment:
docs/source/requirements.txt should be deleted (it was left over cruft)
instead use `poetry add -D m2r2` which will add the m2r2 package to poetry. Make sure to run `poetry lock` after the add

Reply:
@goodwillcoding will RTD pick up the Poetry config? IIRC RTD relies on requirements.txt.

Reply:
`docs/source/requirements.txt` is now deleted from main. Here are the steps to add this dependency:

1. run `poetry add m2r2 --dev`
2. edit `pyproject.toml`, to move it under the `# docs` section of `[tool.poetry.dev-dependencies]`
3. add its name to the `[tool.poetry.extras]` as well
4. commit `pyproject.toml` and `poetry.lock`

Reply:
Disregard—not needed since the doc is rST. :)